src/layouts/jis109.rs

```diff
@@ -5,6 +5,11 @@ use crate::{DecodedKey, HandleControl, KeyCode, KeyboardLayout, Modifiers};
 /// A standard Japan 106-key (or 109-key including Windows keys) keyboard.
 ///
 /// Has a small space bar, to fit in extra keys.
+///
+/// Note this doesn't currently work with Scancode Set 2, because I don't know
+/// what the Set 2 values are for the extra JIS-109 specific keys. Also, you're
```

Comment:
You list Scancode Set 2 values for Oem9-Oem13 in the conversion table in README.md.

Reply:
Nah. There's always something.